fedimint-rocksdb/src/lib.rs

```diff
@@ -159,6 +161,28 @@ fn next_prefix(prefix: &[u8]) -> Option<Vec<u8>> {
 impl IRawDatabase for RocksDb {
     type Transaction<'a> = RocksDbTransaction<'a>;
     async fn begin_transaction<'a>(&'a self) -> RocksDbTransaction {
+        let count = self.1.fetch_add(1, std::sync::atomic::Ordering::Relaxed);
+        if count % 100_000 == 0 {
```

Comment:
This is kinda hacky, do we even know how often that condition will be hit? I'd make it time based.

Reply:
Yeah, this one is purely a hack. I might need to clean it up into a proper abstraction.